addons/website_sale_stock/tests/test_website_sale_stock_multilang.py

```diff
@@ -5,25 +5,24 @@
 from odoo.tests import tagged
 from odoo.tests.common import HttpCase
 
+from odoo.addons.website_sale_stock.tests.common import WebsiteSaleStockCommon
+
 
 @tagged('post_install', '-at_install')
-class TestWebsiteSaleStockMultilang(HttpCase):
+class TestWebsiteSaleStockMultilang(HttpCase, WebsiteSaleStockCommon):
```

Comment:
I'm not sure using the common here is worth the cost :thinking: 

You're only using it to simplify one call to `create` and adding the common adds default data that is not used at all here :thinking:.
I believe that sometimes, using the common everywhere is not worth the cost (at least for the moment, until we introduce a better way to speed up setups).

Or you could merge the 4 different `HttpCase` test classes of `website_sale_stock` into one unique test class.  It's a bit meaningless to have 4 separate classes each having only 1 or 2 tests.  That way, introducing the common once for those 4 tests could be worth it (and only initialize the test data once, not 4 times).

Reply:
I hesitated if I should inherit or not, I removed the common but not sure if we should merge it all together, it is not our scope, maybe it is their internal guidelines?

Reply:
I doubt it's their guidelines, but it's ok to limit the changes in other code if you prefer :ok_hand: 
You can remove the common from `TestWebsiteSaleStockProductConfigurator` too then :eyes: